### PR TITLE
Initial add volunteer form

### PR DESCRIPTION
**Pull request by DavidRez**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ module.exports = {
         include : SRC_DIR,
         loader : 'babel-loader',      
         query: {
-          presets: ['react', 'es2015']
+          presets: ['react', 'es2015','stage-0']
```
Comment (codydaig) [inline webpack.config.js]: Just curious, what does stage-0 do?

File: client/components/Form.jsx [hunk 2]
```diff
@@ -0,0 +1,188 @@
+import React from "react";
```
Comment (codydaig) [inline client/components/Form.jsx]: Can we call this file something other than "Form". This is the AddVolunteer Form, right? So I'd call it something along the lines of `AddVolunteer` or something more descriptive than Form.

File: client/components/Email.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```
Comment (codydaig) [inline client/components/Email.jsx]: This file seems to be indented with 4 spaces. It should be 2.

File: client/components/Form.jsx [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+      },
+      classes: ["Probationary", "Trainee", "Apprentice", "General"],
+      startDate: new Date()
+    };
+  }
+  componentDidMount () {
+    this.handleStart(new Date())
+  }
+
+  handleFirst(e) {
```
Comment (codydaig) [inline client/components/Form.jsx]: We can make these dynamic where we don't have to have a function for each element. 
https://reactjs.org/docs/forms.html#handling-multiple-inputs

File: client/components/Input.jsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```
Comment (codydaig) [inline client/components/Input.jsx]: This file also seems to have 4 space indentation.

File: client/components/Select.jsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```
Comment (codydaig) [inline client/components/Select.jsx]: This file has a mix of indentations.

File: webpack.config.js (hunk 1 above)
Comment (DavidRez) [reply webpack.config.js]: reads the spread operator i'm using when changing specific properties of the newVolunteer state

File: client/components/Form.jsx (hunk 2 above)
Comment (DavidRez) [reply client/components/Form.jsx]: renaming it "AddVolunteerForm"

File: webpack.config.js (hunk 1 above)
Comment (DavidRez) [reply webpack.config.js]: yes, they are supposed to be reusable. ill create a folder for elements.

**Pull request merged**